trusted-committer-article.md

```diff
@@ -0,0 +1,295 @@
+level of risk associated with them.
+
+The Product Owner is the third role in InnerSource. Whether or not there is one
+depends on the type of InnerSource community and the products they build. 
+Similar to agile processes, the PO is responsible for defining and prioritizing
+requirements and stories for the community to implement. The PO therefore often
+interacts with the TC, e. g. in making sure that a requested or contributed
+feature actually belongs to the product. In the Host/Guest analogy, the PO
+would be the one of determines what Host and Guest work on in the communities
+House. Please check out our Product Owner Learning Path video for more detailed
+information.
+
+---(not added in text, yet)
+- Product Owner: Distinction between the guest and host PO - have to negotiate
+  requirements, if necessary.
+---
+
+## Why role names matter
+
+Let's spend a few moments and discuss why the naming - "Trusted Committer" - is
+really important. The role of the TC is present in every successful InnerSource
+community but not every community uses that name. That was actually true for
+myself in my InnerSource career - I used the terms Maintainer.  But it turns
+out that conficts with the technical role "Maintainer" defined by GitHub, for
+instance.  Apache, e. g., uses the term _Committer_, too, but they attaches
+fewer and mostly tech oriented responsibilities to that role. The TC role with
+its additional community oriented responsibilities goes beyond that. The
+"Trusted" in TC conveys that they are trusted and thus empowered by both their
+management and their community to do their job. By fostering openness and
+transparency, TCs builds trust in the process and also the product being built.
+
+So similar to how naming is important in writing software, it is, too, for the
+roles in an InnerSource project so that everyone has the same notion of the
+role and is aware what can be expected from s. o. with that role.
+
+## Ensuring Product Quality
+
+- define quality goals
+    - many facets: conceptual integrity, cleanliness of the code, test coverage
+      and other goals
+- ensure that decisions regarding quality are followed through
+- own tech related decisions 
+    - doesn't mean the TC does all the work or mandates what others are doing
+    - TC just needs to be sure the work is done in the end
+    - even if decisions are not made by her, needs to advocate and, if
+      necessary defend them them outside the community.
+- act as stakeholder of the SW/Architecture towards rest of org
+- set an example
+- make sure community has what it needs to produce quality sw
+    - communicated quality goals/standards
+    - empowerment by management (e. g. for necessary refactorings)
+    - the right tools
+    - prioritize quality and release dates
+- ensure quality at strategic level (longevity of the SW)
+- review and accept contributions
+    - Generally speaking, the TCs role is not to accept all contributions but
```

Comment:
And anything that does not meet that criteria the TC helps and encourages (within reason) the contributor to alter the submission so that it is so-acceptable.